src/main/java/com/pusher/client/channel/PrivateEncryptedChannel.java

```diff
@@ -0,0 +1,11 @@
+package com.pusher.client.channel;
+
+/**
+ * Represents a subscription to an encrypted private channel.
+ */
+public interface PrivateEncryptedChannel extends Channel {
+
+/**
+ * it's not currently possible to send a message using encrypted private channels
+ */
```

Comment:
```suggestion
    // it's not currently possible to send a message using private encrypted channels
```
no need for Javadoc style comment here (also "private encrypted" order consistency)